pkg/crc/machine/machine.go

```diff
@@ -262,6 +275,11 @@ func (client *client) Start(startConfig StartConfig) (*StartResult, error) {
 		return nil, errors.Wrap(err, "Error updating public key")
 	}
 
+	// Trigger disk resize, this will be a no-op if no disk size change is needed
+	if _, err = sshRunner.Run("sudo xfs_growfs / >/dev/null"); err != nil {
```

Comment:
Does it work with OKD?

Reply:
@guillaumerose yes it should work with OKD also.